e2e/cypress/integration/enterprise/system_console/system_scheme_permission_spec.js

```diff
@@ -93,10 +93,7 @@ const removePermission = (permissionCheckBoxTestId) => {
 // # If enabled is true assumes the user has the permission enabled and checks for no system message
 const channelMentionsPermissionCheck = (enabled) => {
     // # Type @here and post it to the channel
-    cy.get('#post_textbox').clear().type('@here{enter}');
-
-    // # Wait for system message to appear
-    cy.wait(500); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
If system message is expected, can we add `cy.uiWaitUntilMessagePostedIncludes`?

Reply:
The verification happens in the test itself and is conditional, so that won't be appropriate.